scripts/fast-reboot

```diff
@@ -324,7 +324,7 @@ function reboot_pre_check()
     fi
     
     # Make sure ASIC configuration has not changed between images
-    ASIC_CONFIG_CHECK_SCRIPT="/usr/bin/asic_config_check"
+    ASIC_CONFIG_CHECK_SCRIPT="/usr/local/bin/asic_config_check"
```

Comment:
is it doable not to change the path?

Reply:
Unfortunately, no. Wheels install to `/usr/local/bin`, whereas .debs install to `/usr/bin`. There's no way to change this behavior.